telethon/tl/custom/button.py

```diff
@@ -266,8 +268,36 @@ def force_reply(single_use=None, selective=None, placeholder=None):
         Args:
             placeholder (str):
                 text to show the user at typing place of message.
+                Telegram crops the placeholder and shows only only
+                64 Characters and adds ellipsis (…) character as 65th.
         """
         return types.ReplyKeyboardForceReply(
             single_use=single_use,
             selective=selective,
             placeholder=placeholder)
+
+    @staticmethod
+    def buy(text):
+        """
+        Creates an Inline button to Buy a Product.
+        It can be only used with Invoice.
+        It should be the first button of the Invoice.
+        On Not Specifying Button, Telegram will Automatically add button to That Invoice.
+        Read More - https://core.telegram.org/api/payments
+
+        Args:
+            text (str):
+                text to show on the Buy Button.
```

Comment:
```suggestion
        Creates a new inline button to buy a product.

        This can only be used when sending files of type
        :tl:`InputMediaInvoice`, and must be the first button.

        If the button is not specified, Telegram will automatically
        add the button to the message. See the
        `Payments API <https://core.telegram.org/api/payments>`__
        documentation for more information.
```